books/refgram/src/logic/explicit_binding.md

```diff
@@ -3,7 +3,7 @@
 ## Left atom place selection
 
 When the speaker wants to interact with an argument that normal chaining doesn't
-select (due to transitivity, or to interact with O or U arguments), it is
+select (due to transitivity, or to interacting with O or U arguments), it is
```

Comment:
I don't understand the change.
"[when the speaker wants] to interact with O or U arguments]"